app/models/groupe_instructeur.rb

```diff
@@ -74,7 +74,7 @@ def routing_to_configure?
 
   def invalid_rule?
```

Comment:
Version pédante avec du pattern matching mais avec moins de `!` , Il faut importer le namespace `Logic` au passage

```suggestion
  def invalid_rule?
    !valid_rule
  end
  
  def valid_rule?
    [routing_rule.left, routing_rule, routing_rule.right] in [ChampValue, Eq | NotEq, Constant] &&
      routing_rule_matches_tdc?
  end
```

Reply:
ha ouais, joli ! :sunglasses: